public/app/views/personal-info/prospect-form.component.html

```diff
@@ -313,10 +313,50 @@ <h3 class="subheading text-center">Now the legal details. Please read through th
                   <ul>
-                    <li><a href="https://home.capitalone360.com/terms_360savings#Truth-in-Savings-Disclosure" target="_blank" class="ng-binding">Truth in Savings Disclosures</a></li>
-                    <li><a href="https://home.capitalone360.com/terms_360savings#Account-Terms-and-Conditions" target="_blank" class="ng-binding">Account Terms and Conditions</a></li>
-                    <li><a href="https://home.capitalone360.com/terms_360savings#Electronic-Fund-Transfer-Disclosure" target="_blank" class="ng-binding">Electronic Fund Transfer Disclosure</a></li>
-                    <li><a href="https://home.capitalone360.com/terms_360savings#Privacy-Policy" target="_blank" class="ng-binding">Privacy Policy</a></li>
+                    <div [ngSwitch]="productId">
+                      <div *ngSwitchCase="'3000'">
+                        <li>
+                          <a href="https://home.capitalone360.com/terms_360savings#Truth-in-Savings-Disclosure" target="_blank" class="ng-binding">Truth in Savings Disclosures</a>
+                        </li>
+                        <li>
+                          <a href="https://home.capitalone360.com/terms_360savings#Account-Terms-and-Conditions" target="_blank" class="ng-binding">Account Terms and Conditions</a>
+                        </li>
+                        <li>
+                          <a href="https://home.capitalone360.com/terms_360savings#Electronic-Fund-Transfer-Disclosure" target="_blank" class="ng-binding">Electronic Fund Transfer Disclosure</a>
+                        </li>
+                        <li>
+                          <a href="https://home.capitalone360.com/terms_360savings#Privacy-Policy" target="_blank" class="ng-binding">Privacy Policy</a>
+                        </li>
+                      </div>
+                      <div *ngSwitchCase="'3300'">
+                        <li>
+                          <a href="https://www.capitalone.com/online-money-market-account/disclosures/" target="_blank" class="ng-binding">Money Market Disclosures</a>
+                        </li>
+                        <li>
+                          <a href="https://www.capitalone.com/online-money-market-account/disclosures/#360moneymarketagreement" target="_blank" class="ng-binding">Account Terms and Conditions</a>
+                        </li>
+                        <li>
+                          <a href="https://www.capitalone.com/online-money-market-account/disclosures/#electronicfundtransferdisclosurestatement" target="_blank" class="ng-binding">Electronic Fund Transfer Disclosure</a>
+                        </li>
+                        <li>
+                          <a href="https://www.capitalone.com/online-money-market-account/disclosures/#privacypolicy" target="_blank" class="ng-binding">Privacy Policy</a>
+                        </li>
+                      </div>
+                      <div *ngSwitchDefault>
```

Comment:
Isn't productId 3000 the default? Why not just an `if` here to reduce the duplicated code?